### PR TITLE
ISPN-9616 server does not include multimap dependency

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-9616

### CODE REVIEW

**Comment by karesti** [comment]
@tristantarrant

**Comment by karesti** [comment]
@tristantarrant done, adding an additional IT test for counters

**Comment by karesti** [comment]
3 unrelated test failing

File: feature-pack/embedded/src/main/resources/modules/org/infinispan/multimap/slot/module.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module xmlns="urn:jboss:module:1.5" name="org.infinispan.multimap" slot="@infinispan.module.slot@">
+    <resources>
+        <artifact name="${org.infinispan:infinispan-multimap}"/>
+    </resources>
+
+    <dependencies>
+        <module name="javax.api"/>
+        <module name="javax.transaction.api"/>
```
[the remark this answers is not in the document]
Comment (karesti) [reply feature-pack/embedded/src/main/resources/modules/org/infinispan/multimap/slot/module.xml]: yes, I do use the API

**Comment by ryanemerson** [comment]
@karesti Can you update the second commit to include the JIRA id and have a more meaningful description.

**Comment by karesti** [comment]
@ryanemerson I did not add the number because I added a counter's one too haha but allright

**Comment by karesti** [comment]
@ryanemerson done, feel free to merge ❤️

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**